src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -346,13 +353,27 @@ final class OnboardingInfoViewController: UIViewController {
 			return true
 		}
 
-		guard !exposureManagerActivated else {
+		guard store.exposureActivationOnboardingState == .notActivated else {
 			completion?()
 			return
 		}
 
+		// we need to disable back navigation until we have an
+		// response for activation. Otherwise we can navigate back,
+		// kill the closure and never receive a result. When next
+		// activation request gets triggered an alter will appear
```

Comment:
Do you mean "alter" or is this a typo and should be "altert"? :D

Reply:
Thanks for the information, I fixed the typo